packages/node/package.json

```diff
@@ -59,9 +61,11 @@
     "etag": "1.8.1",
     "execa": "3.2.0",
     "fs-extra": "11.1.0",
+    "jest": "29.5.0",
```

Comment:
Why was jest added here? Can we use the same version as the other packages?

Reply:
I added `jest` here because I was fighting ESM support. `ts-jest` didn't support `.mts` files, so I updated that with `jest`. I was thinking of updating the top-level `jest`, but saw that `error-utils` has a different version of `jest` (https://github.com/vercel/vercel/commit/49f453742b190c989447bedf3c85947287086688#diff-c01b191e54a63156af3b3374a1a503629e48ea18b3b03b56d0dbea76ef867a23R22), so I went that route.

Before I finish this PR, I'll try updating the top-level `jest` so we can remove `jest` from `error-utils` and `node`.